app/Http/Controllers/BeatmapsController.php

```diff
@@ -67,8 +67,13 @@ public function scores($id)
         $query->withMods($mods);
         $query->withType($type, compact('user'));
 
+        $includes = ['user', 'user.country'];
+        if ($user !== null) {
+            $includes[] = "userReportPresence:user_id($user->user_id)";
```

Comment:
Probably better to return a list of reported score ids for the current beatmap as a top level property in `$results` instead of trying to inline the state.